FlaxEditor/Surface/VisjectSurface.Input.cs

```diff
@@ -452,15 +446,19 @@ public override bool OnKeyDown(Keys key)
                 case Keys.A:
                     SelectAll();
                     return true;
+
                 case Keys.C:
                     Copy();
                     return true;
+
                 case Keys.V:
                     Paste();
                     return true;
+
```

Comment:
I'm sorry for those lines, my auto-formatter was overly active and I missed it when committing the code.

Reply:
That's fine